### ISSUE TITLE
fix erlotnib bug and pin erlotnib version

### ISSUE THREAD

**Issue by martinjrobins** [pkpdapp-team/pkpdapp#95]
deployment is failing due to erlotnib.

Looks like its due to changes in erlotnib api. @DavAug, can you fix this and then pin the version of erlotnib used to a specific commit?

```
Traceback (most recent call last):
  File "/pkpdapp/pkpdapp/manage.py", line 29, in <module>
    main()
  File "/pkpdapp/pkpdapp/manage.py", line 25, in main
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 377, in execute
    django.setup()
  File "/usr/local/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.6/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python3.6/site-packages/django/apps/config.py", line 90, in create
    module = import_module(entry)
  File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 941, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/pkpdapp/pkpdapp/simulate/apps.py", line 9, in <module>
    import erlotinib as erlo
  File "/usr/local/lib/python3.6/site-packages/erlotinib/__init__.py", line 8, in <module>
    from . import (  # noqa
  File "/usr/local/lib/python3.6/site-packages/erlotinib/apps/__init__.py", line 12, in <module>
    from ._simulation import (  # noqa
  File "/usr/local/lib/python3.6/site-packages/erlotinib/apps/_simulation.py", line 17, in <module>
    import erlotinib.apps as apps
AttributeError: module 'erlotinib' has no attribute 'apps'
The command '/bin/sh -c python /pkpdapp/pkpdapp/manage.py migrate --noinput' returned a non-zero code: 1
```

**Comment by DavAug** [comment]
> deployment is failing due to erlotnib.
> 
> Looks like its due to changes in erlotnib api. @DavAug, can you fix this and then pin the version of erlotnib used to a specific commit?
> 
> ```
> Traceback (most recent call last):
>   File "/pkpdapp/pkpdapp/manage.py", line 29, in <module>
>     main()
>   File "/pkpdapp/pkpdapp/manage.py", line 25, in main
>     execute_from_command_line(sys.argv)
>   File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
>     utility.execute()
>   File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 377, in execute
>     django.setup()
>   File "/usr/local/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
>     apps.populate(settings.INSTALLED_APPS)
>   File "/usr/local/lib/python3.6/site-packages/django/apps/registry.py", line 91, in populate
>     app_config = AppConfig.create(entry)
>   File "/usr/local/lib/python3.6/site-packages/django/apps/config.py", line 90, in create
>     module = import_module(entry)
>   File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
>     return _bootstrap._gcd_import(name[level:], package, level)
>   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
>   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
>   File "<frozen importlib._bootstrap>", line 941, in _find_and_load_unlocked
>   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
>   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
>   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
>   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
>   File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
>   File "<frozen importlib._bootstrap_external>", line 678, in exec_module
>   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
>   File "/pkpdapp/pkpdapp/simulate/apps.py", line 9, in <module>
>     import erlotinib as erlo
>   File "/usr/local/lib/python3.6/site-packages/erlotinib/__init__.py", line 8, in <module>
>     from . import (  # noqa
>   File "/usr/local/lib/python3.6/site-packages/erlotinib/apps/__init__.py", line 12, in <module>
>     from ._simulation import (  # noqa
>   File "/usr/local/lib/python3.6/site-packages/erlotinib/apps/_simulation.py", line 17, in <module>
>     import erlotinib.apps as apps
> AttributeError: module 'erlotinib' has no attribute 'apps'
> The command '/bin/sh -c python /pkpdapp/pkpdapp/manage.py migrate --noinput' returned a non-zero code: 1
> ```

That is very weird! It runs on my machine, even when I reinstalled the pkpdapp and erlotinib in my venv again...

**Comment by DavAug** [comment]
And the apps module is definitely there? Is it possible that this is a problem with python 3.6, i.e. the module import handling? I am using python 3.7

**Comment by martinjrobins** [comment]
Fixed via #96
